dev-packages/node-integration-tests/suites/public-api/LocalVariables/test.ts

```diff
@@ -80,8 +80,8 @@ conditionalTest({ min: 18 })('LocalVariables integration', () => {
     child.on('message', msg => {
       reportedCount++;
       const rssMb = msg.memUsage.rss / 1024 / 1024;
-      // We shouldn't use more than 100MB of memory
-      expect(rssMb).toBeLessThan(100);
+      // We shouldn't use more than 120MB of memory
```

Comment:
@timfish not sure how bad that is, is that just a "random" number or is the fact that this is exceeding this here a sign that it is actually leaking memory? 🤔 I would expect this to use _more_ memory now if it's OTEL powered than before, but 🤷

Reply:
If memory is leaked, it exceeds this quite quickly. 100MB was chosen simply because it was the lowest figure where the tests would pass reliably.